db/seeds.rb

```diff
@@ -5,3 +5,183 @@
+    prep_time: 10,
+    cook_time: 0,
+    description: 'Refreshing Caprese salad with ripe tomatoes, mozzarella, and basil.',
+    notes: 'Use high-quality balsamic glaze for the best flavor.',
+    rating: 4.4,
+    is_favorite: true
+  },
+  {
+    name: 'Vegetable Curry',
+    yield: '4 servings',
+    ingredients: "Mixed vegetables (e.g., carrots, peas, potatoes),\nCurry paste,\nCoconut milk,\nOnion,\nGarlic,\nGinger",
+    directions: "1. Saute onion, garlic, and ginger in curry paste.\n2. Add mixed vegetables and saute until coated.\n3. Pour in coconut milk and simmer until vegetables are tender.",
+    prep_time: 25,
+    cook_time: 30,
+    description: 'Flavorful vegetable curry with a creamy coconut milk base.',
+    notes: 'Serve over rice or with naan bread.',
+    rating: 4.2,
+    is_favorite: false
+  },
+  {
+    name: 'Homemade Macaroni and Cheese',
+    yield: '4 servings',
+    ingredients: "Elbow macaroni,\nCheddar cheese,\nButter,\nFlour,\nMilk,\nMustard,\nSalt,\nPepper",
+    directions: "1. Cook elbow macaroni according to package instructions.\n2. Make a roux with butter and flour.\n3. Gradually whisk in milk and bring to a simmer.\n4. Stir in shredded cheddar cheese until melted.\n5. Season with mustard, salt, and pepper.\n6. Mix with cooked macaroni.",
+    prep_time: 20,
+    cook_time: 25,
+    description: 'Classic macaroni and cheese made from scratch with a creamy cheese sauce.',
+    notes: 'Bake with breadcrumbs for a crunchy topping.',
+    rating: 4.6,
+    is_favorite: true
+  },
+  {
+    name: 'Lemon Garlic Roast Chicken',
+    yield: '4 servings',
+    ingredients: "Whole chicken,\nLemon,\nGarlic,\nRosemary,\nThyme,\nOlive oil,\nSalt,\nPepper",
+    directions: "1. Preheat oven to 375°F (190°C).\n2. Rub chicken with olive oil, salt, and pepper.\n3. Stuff cavity with lemon halves, garlic, rosemary, and thyme.\n4. Roast until internal temperature reaches 165°F (74°C).",
+    prep_time: 15,
+    cook_time: 90,
+    description: 'Juicy and flavorful roast chicken with a zesty lemon and garlic infusion.',
+    notes: 'Let the chicken rest before carving for best results.',
+    rating: 4.7,
+    is_favorite: true
+  },
+  {
+    name: 'Quinoa Salad',
+    yield: '3 servings',
+    ingredients: "Quinoa,\nCucumber,\nCherry tomatoes,\nRed onion,\nFeta cheese,\nKalamata olives,\nOlive oil,\nLemon juice",
+    directions: "1. Rinse quinoa and cook according to package instructions.\n2. Chop cucumber, cherry tomatoes, red onion, and olives.\n3. Mix quinoa with vegetables and crumbled feta cheese.\n4. Drizzle with olive oil and lemon juice.",
+    prep_time: 15,
+    cook_time: 20,
+    description: 'Healthy and refreshing quinoa salad with Mediterranean flavors.',
+    notes: 'Add grilled chicken for a protein boost.',
+    rating: 4.2,
+    is_favorite: false
+  },
+]
+
+recipes_data.each do |recipe_data|
+  Recipe.create!(recipe_data)
+end
```

Comment:
I'd like to be able to run the seeds multiple times without getting errors or duplicate data. The following should find a recipe with the same name and update it or create a new one.

```suggestion
recipes_data.each do |recipe_data|
  Recipe.find_or_initialize_by(name: recipe_data[:name]).update(recipe_data)
  end
end
```